localisation/00_wc_government.csv

```diff
@@ -1,23 +1,21 @@
+black_pact_government_desc;The leaders of this government have drawn up a black pact with the Old Gods. Their driving purpose is to bring about the Hour of Twilight, free the Old Gods from their prisons, resurrect their Black Empire, and ultimately end all life on Azeroth. In line with bringing about these prophesied final moments of existence, followers of this black pact with the Old Gods do not find diplomacy effective and choose violence and conflict to further their goals;La volont� des dieux tr�s anciens est la base de ce Pacte avec les Forces des T�n�bres, qui apporte douleur, folie et r�pand la parole de ses sombres ma�tres � travers tout Azeroth. Ils n'ont aucun principes, seulement l'ambition de se rapprocher de la Fin des temps, le moment o� Azeroth va plonger dans la d�solation. Ils ne savent ce qu'est la diplomatie, ni une vie paisible.;Der Wille der alten G�tter leitet diesen schwarzen Pakt, nichts als Schmerz, Wahnsinn und das Wort seiner Herren in die L�nder Azeroth zu tragen. Sie haben keine anderen Prinzipien, als die Endzeit, den Augenblick wenn Azeroth der Dunkelheit verf�llt, herbeizuf�hren. Mit dieser Ambition kennen sie weder Diplomatie noch ein friedliches Miteinander.;Der Wille der alten G�tter leitet diesen schwarzen Pakt, nichts als Schmerz, Wahnsinn und das Wort seiner Herren in die L�nder Azeroth zu tragen. Sie haben keine anderen Prinzipien, als die Endzeit, den Augenblick wenn Azeroth der Dunkelheit verf�llt, herbeizuf�hren. Mit dieser Ambition kennen sie weder Diplomatie noch ein friedliches Miteinander.;;;;;;;;;;x
 dragon_government;Dragonflight;Vol draconique;Drachenschwarm;;;;;;;;;;;x
-dragon_government_desc;A dragonflight is a hierarchical society of dragons and other dragonkin.;Un vol draconique est une soci�t� hi�rarchique compos�e de dragons et autres draconiens.;Der Drachenschwarm ist eine hierarchische Gesellschaft von Drachen und Drachenartigen.;;;;;;;;;;;x
+dragon_government_desc;The different races of dragons are split up between different dragonflights. Each flight is led by a Dragon Aspect who holds dominion over all the brethren, originally chosen by the Titans. The primary great flights are the blue, black, bronze, green, and red dragonflights. Other lesser flights exist that of which are considered artificial or unnatural, such as the Infinite or Chomatic dragonflights. ;Un vol draconique est une soci�t� hi�rarchique compos�e de dragons et autres draconiens.;Der Drachenschwarm ist eine hierarchische Gesellschaft von Drachen und Drachenartigen.;;;;;;;;;;;x
```

Comment:
> Each flight is led by a Dragon Aspect who holds dominion over all the brethren

What about independent dragons

<details>

![image](https://user-images.githubusercontent.com/46576860/64912827-6640a400-d73d-11e9-8375-8b8417a63385.png)

</details>

> The different races of dragons are split up between different dragonflights.

or dragons that have other dragons as vassals

<details>

![image](https://user-images.githubusercontent.com/46576860/64912853-e535dc80-d73d-11e9-9b6e-0f547fb62ec7.png)

</details>

Reply:
Pretty sure uncolonized_government is not a thing anymore, as I recall us removing all of those some time ago.